src/mmw/js/src/modeling/views.js

```diff
@@ -457,14 +457,13 @@ var ScenarioDropDownMenuOptionsView = Marionette.ItemView.extend({
     renameScenario: function() {
         var self = this,
             collection = self.model.collection,
-            validate = _.bind(collection.validateNewScenarioName, collection),
-            curriedValidationFunction = _.curry(validate)(this.model),
+            validate = _.bind(collection.validateNewScenarioName, self.model),
```

Comment:
Don't think we should change it, but could we have also moved this method onto the `ScenarioModel` and avoided the binding?

Reply:
Yeah, totally. I think that would work as well.